TransferMethodRepository/Sources/TransferMethodConfigurationRepository.swift

```diff
@@ -53,7 +64,8 @@ public final class RemoteTransferMethodConfigurationRepository: TransferMethodCo
     private var transferMethodConfigurationFieldsDictionary =
         [HyperwalletTransferMethodConfigurationFieldQuery: HyperwalletTransferMethodConfigurationField]()
     private var transferMethodConfigurationKeys: HyperwalletTransferMethodConfigurationKey?
-
+    private var transferMethodFeesAndProcessingTime =
```

Comment:
transferMethodFeesAndProcessingTime -> transferMethodTypeFeesAndProcessingTime

Reply:
Removed cacheing of Transfer method types, fee and processing time API